docs/_data/main-nav.yaml

```
@@ -137,4 +137,6 @@ toc:
     collapse-id: kong-products
     items:
       - title: Publish API to Dev Portal
-        url: /insomnia/publish-to-dev-portal
\ No newline at end of file
+        url: /insomnia/publish-to-dev-portal
+      - title: Declarative Config (decK)
```

```suggestion
      - title: Kong Declarative Config (for use with decK)
```

Something like that? `decK` is just used as a way to push declarative config into Kong